docs/framework-integration/react.md

```diff
@@ -10,356 +10,287 @@ tags:
 </header>
 
 {% band header="Using PatternFly Elements in your React app" %}
-  To get web components to work with React it’s pretty easy and straightforward. If you’d like to follow along,
-  go ahead and [create a new React CodeSandbox on codesandbox.io](https://codesandbox.io/s/new).
-  The React sandbox uses [create-react-app](https://github.com/facebook/create-react-app) to scaffold an app and you can
-  view your changes in real-time right in the web app. With CodeSandbox, you can also add any npm dependency with
-  just a few button clicks. If you want to run this app locally, you can
-  [clone the repository on 
-  GitHub](https://github.com/kylebuch8/patternfly-elements-with-react).
-
-  “Using PatternFly Elements in your React App” is broken down into four sections:
-  - Initial setup
-  - Adding PatternFly Elements
-  - Interacting with our web components API
-  - Adding icing on the cake
-
-  Each section will show you exactly what you need to do with code snippets and an accompanying CodeSandbox that you can edit or fork.
+We created React wrapper for each web component using [@lit/react](https://github.com/lit/lit/tree/main/packages/react#litreact) library. This library helps us make the React wrapper for web components, and we can use it like React components. Please check this [example](https://codesandbox.io/p/sandbox/pfe-react-wrappers-3g6x6r?file=%2Fpackage.json%3A1%2C1) for more details. If you are more interested in how we are doing in the patternfly element library. Please check this [library](https://www.npmjs.com/package/@lit-labs/gen-wrapper-react).
+
+“Using PatternFly Elements in your React App” is broken down into four sections:
+
+- Initial setup
+- Adding PatternFly Elements
+- Add Switch (pf-switch) component
+- Add Tooltip (pf-tooltip) component
+- Interacting with our web components API
+- Codesandbox example
+- Github repo
+- Why does React need special treatment?
+- Recap
+
+Each section will show you exactly what you need to do with code snippets.
 {% endband %}
 
 {% band header="Initial setup" %}
-  Import React and ReactDOM at the top of the `index.js` file in the `/src/` directory.
+Create a new React project with [Vite](https://vitejs.dev/guide/#scaffolding-your-first-vite-project) by running this command
```

Comment:
I'm not sure we'd want to focus on a single bundle tool here.  If anything maybe we should have this section contain a `pf-tabs` and give instructions for the vanilla and other popular tools.

Reply:
Agreed, we should start from import

We will have to document various gotchas and workarounds with all the wonderful, faultless, and totally rad opinionated and forward-breaking tooling that's grown up around react, not unlike a festering pustule...

But we'll have to do that in a separate page

Reply:
@zeroedin @bennypowers 
Regarding multiple tools -> 
I think we can add one note to the document something like this:
Note: We have provided an example using the Vite tool. You can also try using other tools, such as Webpack.
What do you think?

Regarding various examples of components:
We can provide framework examples for each component, similar to the popover component page: https://patternflyelements.org/components/popover/
This allows users to see how a component can be used in different frameworks.